docs/editions.rst

```diff
@@ -94,31 +115,59 @@ CrateDB Community Edition
 The CrateDB *Community Edition* (CrateDB CE) does not include any
 :ref:`enterprise-features` but can be run on as many nodes as you wish.
 
-CrateDB CE must be built from source, like so:
+.. NOTE::
+
+   CrateDB requires a `Java virtual machine`_ to run.
+
+   Starting with CrateDB 4.2, a JVM is bundled with the tarball and no
+   extra installation is necessary.
+
+   Versions starting from 3.0 to 4.1 require a `Java 11`_ installation. We
+   recommend using `Oracle's Java`_ on macOS and OpenJDK_ on Linux Systems.
+
+   Earlier versions required Java 8.
+
+CrateDB CE must be built from source:
 
 .. code-block:: console
 
    sh$ git clone https://github.com/crate/crate
    sh$ cd crate
-   sh$ git submodule update --init
    sh$ git checkout <TAG>
    sh$ ./gradlew clean communityEditionDistTar
 
-Here, replace ``<TAG>`` with the commit hash of the Git tag that corresponds to
-the `release`_ you wish to use.
+The steps above:
+
+- clone the CrateDB Git repository and navigate into the directory
 
-When the ``gradlew`` command successfully completes, the relevant CrateDB CE
+- replace ``<TAG>`` with the commit hash of the Git tag that corresponds to
+  the `release`_ you wish to use or just the version number of the release
+
+- execute the `Gradle Wrapper`_ script included in the repository to clean up
+  any old build files and to invoke a `distribution plugin`_ that will generate
+  Gradle tasks to build the Community Edition tar archive 
```

Comment:
Not sure if the details about the gradle tasks generation are necessary to mention. That's kind of describing gradle internals. For users the more important piece is that is ends up generating the tarball